recipe/meta.yaml

```
@@ -1,61 +1,211 @@
 {% set name = "streamlit" %}
-{% set version = "1.41.1" %}
+{% set version = "1.42.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6626d32b098ba1458b71eebdd634c62af2dd876380e59c4b6a1e828a39d62d69
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 8c48494ccfad33e7d0bc5873151800b203cb71203bfd42bc7418940710ca4970
+    folder: streamlit
+  - url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: d13d8ed5f0b8b429febe0868619d7b42298e7badba266ee7fa73823039ce7de8
+    folder: streamlit_tests
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  entry_points:
-    - streamlit = streamlit.web.cli:main
-  skip: true  # [s390x or py<38]
+  # Watchdog is not avaliable on win + py313
+  # https://github.com/AnacondaRecipes/streamlit-feedstock/pull/25#discussion_r1951425010
+  # Need to build watchdog >=4.0.2
+  # https://github.com/streamlit/streamlit/blob/1.42.0/lib/setup.py#L143
+  # python_requires=">=3.9, !=3.9.7"
+  skip: True  # [py<39 or py>312]
```

```suggestion
  skip: True  # [py<39]
```
Streamlit is compatible with 3.13.

Either remove `ANACONDA_ROCKET_ENABLE_PY313 : yes` from abs.yaml or build an updated watchdog. I'd prefer we build an updated watchdog.

---

I tried